### ISSUE TITLE
Don't report tags for imported content

### ISSUE THREAD

**Issue by hoschi** [ramitos/jsctags#45]
At the moment jsctags reports tags for imported content, e.g.:
```
import {foo, bar} from './myModule';

export default foo(bar());
```
will report the tags `foo` and `bar`. This leads to the problem that pressing `ctrl+]` on `foo` in line 3, vim will jump to line 1. What I really want is jumping to the definition of foo in './myModule' and not the local declaration of foo. Not reporting the `import` statements would lead to:
* pro
** easy jumping between files
** smaller tag files
* contra
** no listing of declarations from import statements in Tagbar or something like that

In my opinion the pros have more weight than the cons, thoughts?

**Comment by ramitos** [comment]
`foo` and `bar` are still a symbol declared in the scope of that file. IMHO they should be present the tags.

**Comment by hurrtz** [comment]
Agreed, `ctrl-]` in line 3 should bring you to line 1 but `ctrl+]` on line 1 should then lead to myModule.

**Comment by hurrtz** [comment]
Ok, upon further reading:
This is not an issue with jsctags. When you work with `g]` you can see that all relevant tags are there, ready to be used. It is more of a problem with vim and tag navigation. [Heres](https://stackoverflow.com/questions/1054701/get-ctags-in-vim-to-go-to-definition-not-declaration) something to read on that (and some some solutions which work quite well).
